src/lib/formatter/SarifFormatter.ts

```diff
@@ -131,18 +131,19 @@ abstract class SarifFormatter {
 				} else {
 					// Regular violations are placed in the results array
 					// W-8881776: line and column are sometimes strings, but the schema requires them to be numbers
-					const region = {
+					const region: Region = {
 						startLine: parseInt(`${v.line}`),
 						startColumn: parseInt(`${v.column}`)
 					};
 
-					for (const k of ['endLine', 'endColumn']) {
-						if (v[k]) {
-							region[k] = parseInt(v[k]);
-						}
+					if (v.endLine != null) {
```

Comment:
The `if`s make it easier for the linter to recognize that everything here is above board.